CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Unreleased
+
+### Bug fixes and improvements
+
+- Linux .deb packages 'on hold' are now included in the installed software list.
+
```

Comment:
sorry if this wasn't clear in the docs, we manage the changelog adding files into the `changes/` folder as described [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Committing-Changes.md#changes-files)

Reply:
Sorry about that, I read it too fast. I updated the pull request.